chain/types/ethtypes/eth_types.go

```diff
@@ -225,7 +225,8 @@ type EthCall struct {
 	Gas      EthUint64   `json:"gas"`
 	GasPrice EthBigInt   `json:"gasPrice"`
 	Value    EthBigInt   `json:"value"`
-	Data     EthBytes    `json:"data"`
+	Data     *EthBytes   `json:"data"`
+	Input    *EthBytes   `json:"input"`
 }
```

Comment:
We should just write a custom `UnmarshalJSON` function that aliases the field on decode.

Reply:
Alternative version https://github.com/filecoin-project/lotus/pull/11505